### DISCUSSION TITLE
PyTorch 如何加速？

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#590] [Q&A]
【PyTorch】唯快不破：基于Apex的混合精度加速： <https://zhuanlan.zhihu.com/p/79887894>

文档地址：<https://nvidia.github.io/apex/index.html>

**Comment by lartpang** [comment]
可见老夫的收藏！

https://github.com/lartpang/PyTorchTricks
